src/variables/selectors/index.tsx

```diff
@@ -109,7 +109,7 @@ export const getVariablesForDashboard = (state: AppState): Variable[] => {
     Object.values(state.resources.views.byID).filter(
       variable => variable.dashboardID === state.currentDashboard.id
     )
-  )
+  ).sort((a, b) => a.sort_order - b.sort_order)
```

Comment:
what happens with existing variables that don't have a sort order?

Reply:
All variables will now have a sort_order that defaults to 0